### PR TITLE
Automatize "Test Group aggregated reporting data (mSERS) by jurisdiction and epi week"

### PR DESCRIPTION
**Pull request by mp-sgent**

### CODE REVIEW

**Comment by mp-sgent** [comment]
image

**Review by rdutu-vg — APPROVED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java [hunk 1]
```diff
@@ -66,10 +89,43 @@ public MSersDirectorySteps(
...
+        (String disease) -> webDriverHelpers.selectFromCombobox(FACILITY_COMBOBOX, disease));
+    When(
+        "I select ([^\"]*) from Point Of Entry combobox",
+        (String disease) -> webDriverHelpers.selectFromCombobox(POINT_OF_ENTRY_COMBOBOX, disease));
+    When(
+        "I click to Export aggregate report",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(CASE_CONTACT_EXPORT);
+          String fileName = "sormas_aggregate_reports_" + LocalDate.now() + "_.csv";
+          FilesHelper.waitForFileToDownload(fileName, 30);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java:110]: to all above methods, specify also directory location :D

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/mSers/MSersDirectoryPage.java [hunk 2]
```diff
@@ -18,11 +18,24 @@ public class MSersDirectoryPage {
...
   public static final By DISEASE_COMBOBOX = By.cssSelector("#disease div");
+  public static final By POINT_OF_ENTRY_COMBOBOX = By.cssSelector("#pointOfEntry div");
+  public static final By FACILITY_COMBOBOX = By.cssSelector("#healthFacility div");
+  public static final By DISTRICT_COMBOBOX = By.cssSelector("#district div");
+  public static final By REGION_COMBOBOX = By.cssSelector("#region div");
+  public static final By GROUPING_COMBOBOX = By.xpath("(//div[@class='v-filterselect-button'])[1]");
+  public static final By GROUPING_COMBOBOX_INPUT =
+      By.xpath("(//div[@class='v-filterselect-button'])[1]/..//preceding-sibling::input");
+  public static final By SHOW_ROWS_FOR_DISEASES_LABEL =
+      By.xpath("//label[text()=\"Show 0-rows for disease(s)\"]");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/mSers/MSersDirectoryPage.java:29]: can we create it without using text? in this way we can use it also for german environment

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java (hunk 1 above)
Comment (mp-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java:110]: done

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/mSers/MSersDirectoryPage.java (hunk 2 above)
Comment (mp-sgent) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/mSers/MSersDirectoryPage.java]: done

**Pull request merged**